scripts/sign_nuget.bat

```diff
@@ -9,9 +9,9 @@
 SET certFile=%1
 SET certPass=%2
 
-:: Sign all NuGet packages found
+:: Sign all NuGet packages found with our certificate to guarantee authenticity
 @ECHO:
-@ECHO Signing NuGet package with %certFile%...
+@ECHO Authenticating NuGet package with %certFile%...
```

Comment:
I don't know that the changes of `Signing` to `Authenticating` is appropriate. Think of a login system, we authenticate the user against a well-known set of creds that belong to them to make sure the person logging in is who they say they are (but this is an after-the-fact operation) meaning we have their creds already to authenticate against. In this case, we are signing our DLLs so that a system in the future can then authenticate that this came from EasyPost.

Thoughts? This is true of all the changes of `signing` to `authenticating` throughout this PR

Reply:
Fair enough, I'm not a huge fan of the change either. Wanted to clarify the difference between strong-name signing and signing for authenticity like a hash, any synonym suggestions?